android-sdk/defaults/main.yml

```diff
@@ -1,9 +1,6 @@
 ---
 
-project_name: jenkins
 
-rhmap_version: 4.5
-# or
 buildfarm_templates_dir: /path/to/buildfarm_templates_dir
```

Comment:
we should remove `buildfarm_templates_dir`. There is a check later for setting the productised templates dir that only get set if `buildfarm_templates_dir` is undefined.